pkg/cli/powertools/forcesetfield/cmd.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package forcesetfield
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/cli/powertools/diffs"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/cli/powertools/kubecli"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Options configures the behaviour of the ChangeStateIntoSpec operation.
+type Options struct {
+	kubecli.ClusterOptions
+	kubecli.ObjectOptions
+
+	// FieldManager is the field-manager owner value to use when making changes
+	FieldManager string
+
+	// DryRun is true if we should not actually make changes, just print the changes we would make
```

Comment:
If we ship it true I think its dangerous to ever undo that. I'm fine with that but I want to be clear I don't think we should change this after we ship it.

Reply:
Yes, I see your point, I'll fix it up!